client/components/PubEdgeListing/pubEdgeListing.scss

```diff
@@ -16,13 +14,6 @@
         }
     }
 
-    .isolated-controls {
```

Comment:
I removed this (and the `bleed-top-border` mixin) and inlined it into the `PubEdgeListingCard` because I think all `children` of that component should appear as these `isolated-controls` do.